### ISSUE TITLE
Ignore Dates in the Title of the Event

### ISSUE THREAD

**Issue by seeraeubert** [dengste/org-caldav#72]
Hello,

dates in the title of the org item/event should be ignored. Looking into the source code, it seems like the function "org-caldav-change-heading" should do this, but my calender items still contain the date in the title.

Examples:

Org Title | Calender Title
----------- |------------------
Test1 <2015-06-26> | Test1 <2015-06-26>
<2015-06-26> Test2 | <2015-06-26> Test2
Test3 :work: <2015-06-26> | Test3 :work: <2015-06-26>
Test4 <2015-06-26> :work: | Test4 <2015-06-26>

Also note that tags are only ignored if they are at the end of the title, but this is ok for me.

Does this work for anyone else, so I could check if this is only a problem in my specific configuration?

**Comment by hrehfeld** [comment]
Why should they be ignored? Does org-mode do the same?

**Comment by kahironimashte** [comment]
> Does org-mode do the same?

Yes, headings with active timestamps appear in the agenda at the correct date, without the timestap. 

Example:  "* Meeting <2020-07-20 Mon>" is displayed as "Meeting"


This would actually be kind of helpful because using multiple dates in a single ressource is not yet supported (https://github.com/dengste/org-caldav/issues/145). Hiding dates in Headings could be used as a hack instead of a simple list of timestamps, which does not sync to calendars.

Hack:
```
* Meeting
** Meeting <2020-07-19 Sun>
** Meeting <2020-07-20 Mon>
```

Default use in emacs, but org-caldav only syncs first date:
```
* Meeting
 - <2020-07-19 Sun>
 - <2020-07-20 Mon>
```
